src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1430,6 +1987,11 @@ public override IOperation VisitIsNull(IIsNullOperation operation, int? captureI
         {
             throw ExceptionUtilities.Unreachable;
         }
+
+        public override IOperation VisitCaughtException(ICaughtExceptionOperation operation, int? captureIdForResult)
```

Comment:
This shouldn't go in the needs to be implemented region.

Reply:
> This shouldn't go in the needs to be implemented region.

You are correct. Also applies to VisitFlowCapture, VisitFlowCaptureReference and VisitIsNull.

---
In reply to: [167704143](https://github.com/dotnet/roslyn/pull/24770#discussion_r167704143) [](ancestors = 167704143)